src/dbally/data_models/llm_options.py

```diff
@@ -1,6 +1,8 @@
 from dataclasses import asdict, dataclass
 from typing import List, Optional, Union
 
+from openai import NOT_GIVEN, NotGiven
+
 
 @dataclass
 class LLMOptions:
```

Comment:
What is the idea to handle options for different LLMClients when they do not support some parameter?

Reply:
This `LLMOptions` is tied to the OpenAI client, as originally stated in the comments. Each new integration will have its own `LLMOption` class to support a specific API.

Reply:
ahh okay